src/mmw/js/src/draw/views.js

```diff
@@ -148,7 +140,45 @@ var PlaceMarkerView = Marionette.ItemView.extend({
     }
 });
 
-function clearLayer(layer) {
+function changeOutlineLayer(endpoint, tableId, select) {
+    var map = App.getLeafletMap();
+
+    if (endpoint && tableId != undefined) {
+        var ol = App.outlineLayer = new L.TileLayer(endpoint + '.png', {maxZoom: 18});
```

Comment:
Why is maxZoom set to 18? Is that the same as elsewhere in the app? We might want to consider changing this to a setting.